### PR TITLE
#393 Set up logging

### LINKED ISSUE #393: [STORY] Implement a logging tool
## Brief description:
Research and implement a logging tool so that we can track errors within the python api. This logging tool should write to a file when IS_OFFLINE is false and to output in console when IS_OFFLINE is true. Also, it should be extending fastapi's default logging behaviour instead of clashing with it.

### PR DESCRIPTION
**Pull request by IvanObreshkov**
Resolves #393

### CODE REVIEW

File: packages/py-api/py_api/routes/utility_routes.py [hunk 1]
```diff
@@ -1,13 +1,16 @@
+import logging
```
Comment (mbozhilov-qb) [inline packages/py-api/py_api/routes/utility_routes.py:1]: from logging import getLogger

File: packages/py-api/py_api/routes/utility_routes.py [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+from fastapi import APIRouter, Request
 from py_api.utilities.memory import get_current_memory_usage_in_mbs
 
 router = APIRouter()
+logger = logging.getLogger("health")
 
 
 @router.get('/health')
 async def health() -> Dict[str, Any]:
+    logger.warning("test")
```
Comment (mbozhilov-qb) [inline packages/py-api/py_api/routes/utility_routes.py:13]: remove warning

File: packages/py-api/py_api/utilities/parsers.py [hunk 3]
```diff
@@ -6,3 +6,16 @@ async def parse_request_body(body: Callable[..., bytes]) -> Dict[Any, Any]:
...
+    if type(bl) == bool:
+        return bl
+    else:
+        bl = bl.strip().lower()
+        if bl == "true":
+            return True
+        else:
+            return False
+
+        # TODO: Add error handling, possibility of non boolean value
```
Comment (mbozhilov-qb) [inline packages/py-api/py_api/utilities/parsers.py:21]: Remove this comment

File: packages/py-api/py_api/main.py [hunk 4]
```diff
@@ -1,10 +1,11 @@
-from dotenv import load_dotenv  # noqa
+from dotenv import load_dotenv
+from py_api.utilities.logging import get_log_config
+from py_api.utilities.parsers import eval_bool  # noqa
 
 load_dotenv()  # noqa
```
Comment (mbozhilov-qb) [inline packages/py-api/py_api/main.py:5]: Move this after the import of load_dotenv

File: packages/py-api/py_api/routes/utility_routes.py (hunk 1 above)
Comment (IvanObreshkov) [reply packages/py-api/py_api/routes/utility_routes.py]: Done

File: packages/py-api/py_api/routes/utility_routes.py (hunk 2 above)
Comment (IvanObreshkov) [reply packages/py-api/py_api/routes/utility_routes.py]: Done

File: packages/py-api/py_api/utilities/parsers.py (hunk 3 above)
Comment (IvanObreshkov) [reply packages/py-api/py_api/utilities/parsers.py]: Done

File: packages/py-api/py_api/main.py (hunk 4 above)
Comment (IvanObreshkov) [reply packages/py-api/py_api/main.py:3]: Done

**Review by asynchroza — APPROVED** [review]
Thank you for working on this one! Great job! :pill: #bobidroga

**Pull request merged**